R/tab_footnote.R

```diff
@@ -219,6 +219,26 @@ set_footnote.cells_summary <- function(loc, data, footnote) {
   data
 }
 
+set_footnote.cells_grand_summary <- function(loc, data, footnote) {
+
+  rows <- (loc$rows %>% as.character())[-1] %>% as.integer()
```

Comment:
What's going on with `loc$rows` here? I was expecting it to use a resolve of some kind. And isn't it a quosure? I'm confused...

Reply:
This does need some work. With the recent changes from before (with `resolver.R`) I'll now use the `resolve_*()` functions to properly handle all of these cases.

Reply:
Now uses `resolve_vars()`.